### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by li-kai**
As per title. Gonna work on getting the renovate config right asap as this is being a pretty big pain point.

### CODE REVIEW

File: www/package.json [hunk 1]
```diff
@@ -23,30 +23,30 @@
...
+    "@babel/preset-env": "7.4.3",
     "@babel/preset-react": "7.0.0",
     "@babel/preset-typescript": "7.3.3",
-    "@hot-loader/react-dom": "16.8.4",
+    "@hot-loader/react-dom": "16.8.6",
     "@svgr/webpack": "4.1.0",
     "@types/classnames": "2.2.7",
     "@types/enzyme": "3.9.1",
-    "@types/jest": "23.3.13",
+    "@types/jest": "24.0.11",
```
Comment (ZhangYiJiang) [inline www/package.json]: Check if we can delete this since Jest is in Typescript now

**Comment by li-kai** [comment]
Typescript 3.4.2 has a few bugs and 3.5.0 prerelease does not fix them yet. Gonna hold off on the PR for now :/

**Comment by li-kai** [comment]
Most of this fixes should will be split up into their renovate changes.

File: www/package.json (hunk 1 above)
Comment (taneliang) [reply www/package.json]: I couldn't get it to work after removing this

**Pull request closed without merging**